src/futures.rs

```diff
@@ -0,0 +1,962 @@
+use byteorder::ByteOrder;
+use core::future::Future;
+use core::marker::{PhantomData, Unpin};
+use core::mem::size_of;
+use core::pin::Pin;
+use core::task::{Context, Poll};
+use futures_io as io;
+
+pub use byteorder::{BigEndian, LittleEndian, NativeEndian, NetworkEndian};
```

Comment:
This doesn't need to be `pub use`. They should be `pub use` in `src/lib.rs` instead.